kind.sh

```diff
@@ -131,16 +156,20 @@ parse_command_line() {
 
 install_kind() {
     echo 'Installing kind...'
-    curl -sSLo kind "https://github.com/kubernetes-sigs/kind/releases/download/$version/kind-linux-amd64"
-    chmod +x kind
-    sudo mv kind /usr/local/bin/kind
+
+    mkdir -p "$kind_dir"
+
+    curl -sSLo "$kind_dir/kind" "https://github.com/kubernetes-sigs/kind/releases/download/$version/kind-linux-amd64"
```

Comment:
Where is kind_dir set? I see it local to main()

Reply:
yep, variables are inherited

Reply:
Oh nm, I looked at it closer. Those funcs are called by main 👍 Thanks